### ISSUE TITLE
Improve upgrade gateway workflow

### ISSUE THREAD

**Issue by jingtien** [magma/magma#8554]

**Comment by jingtien** [comment]
**Description**
All our partners experience the gateway upgrade workflow in some form.  The current process is unintuitive and tedious.  By making it a more streamlined and a self serve experience, partners would upgrade on their own and more frequently without needing MOSE/PE assistance.  When partners are on the newer versions it lowers the Magma eng cost of continuously supporting older version.

**Possible starter task for Iris**
- [1-pager](https://fb.quip.com/wCZNA089v487) for usability issues when trying to upgrade gateways
- [Screenshots](https://www.figma.com/file/AkW2CmGylqfeTsJaCXbzs1/Gateway-Upgrade?node-id=0%3A1) provided by Hanna
- POCs for research Tim, Arun, Scott, Wally, Mark and Hanna

**Impact**
- Lower number of support tickets
- More likely for partners to be on a later version which allows us to avoid fixing bugs on older versions

**Frequency**
Medium to high, since we always want them to be on a recent version

**Confidence**
Medium, we have a few usability pain points to fix the existing solution

**Eng Effort**
tbd

**Comment by jingtien** [comment]
Waiting on proposal from Tim first

**Comment by andreilee** [comment]
This still remains a requested feature. Awaiting proposal from Tim.

**Comment by shahrahman-fb** [comment]
Closing this issue as part of a repository cleanup effort. This issue has had no activity for over 2 years.

If this issue is still relevant, please reopen it or file a new issue with updated information. We are actively maintaining the project again and want to focus on current, actionable issues.

See: #15833 (Phase 1 cleanup), #15828 (tracking issue)
